build.gradle

```diff
@@ -54,6 +50,11 @@ allprojects {
     apply plugin: 'java-library'
     apply plugin: 'groovy'
 
+    bintraySpinnaker {
+      enabled = true
+      bintrayOrg = 'cfieber'
```

Comment:
not strictly necessary - this extension will have reasonable defaults for our standard services